### DISCUSSION TITLE
Migrate to TMDB api

### DISCUSSION THREAD

**Discussion by bt4ibwem8** [trakt/showly#911] [Ideas / Feature Requests]
Why are you using Trakt api instead of TMDB? I think tmdb is better with more languages and probably also more metadata available...

**Comment by Treveo** [comment]
Showly is using TMDB already for movies and shows data? Trakt.tv is for "tracking".

**Comment by michaldrabik** [comment]
Using both APIs with majority of Trakt being utilized. 

Trakt is also getting all their data from TMDB btw:
https://forums.trakt.tv/t/tv-show-metadata/19076

but since the sync has always been with Trakt and it is more popular service for tracking - it is easier to use. Mainly because TMDB api items do not contain Trakt IDs while Trakt API serves items with both Trakt and TMDB ids.

**Reply by bt4ibwem8** [reply]
But Trakt is probably not supporting all languages from tmdb... So many languages are missing in your app
